code_challenges/code_wars/find_unique_number.py

```diff
@@ -0,0 +1,17 @@
+# https://www.codewars.com/kata/585d7d5adb20cf33cb000235/train/python
+
+
+def find_unique_number(numbers):
+    frequency_of_numbers = {}
+    for single_number in numbers:
+        frequency_of_numbers[single_number] = (
+            frequency_of_numbers.get(single_number, 0) + 1
+        )
+        # if single_number in frequency_of_numbers:
+        #     frequency_of_numbers[single_number] += 1
+        # else:
+        #     frequency_of_numbers[single_number] = 1
```

Comment:
If you want to keep notes around too look back at, I'd recommend setting up a new project and writing them there.  Unused commented out code should be removed.